### DISCUSSION TITLE
基于web端或文本直接作为输入，应怎样接入使用nonebot呢？

### DISCUSSION THREAD

**Discussion by kevinfu1717** [nonebot/discussions#49] [问答] [answered]
想做的效果：对话机器人（并非基于qq或飞书等），通过本文直接作为bot的输入，应该如何接入nonebot呢？

**Comment by yanyongyu** [comment, marked as answer]
你可以自行编写一个adapter来进行事件的构造，或者直接调用 `nonebot.message.handle_event` 并传入 bot、event 对象来触发一次事件处理
